### PR TITLE
Kyle documentation move to .md file, and test install

### PR DESCRIPTION
**Pull request by kylemath**
I moved all the instructions for the notebook into an .md file like a readme and changed the link in main readme to link to it, as well as removed the older .doc files

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -53,4 +53,4 @@ Currently available experiments:
 - SSVEP
 - Visual P300
 - Cueing (Kyle Mathewson)
-- Baseline (Kyle, Eye's Open vs. Closed, needs notebook made)
\ No newline at end of file
+- Baseline (Kyle, Eye's Open vs. Closed, needs notebook made)
```
Comment (kylemath) [inline README.md]: nothing changed here

**Comment by kylemath** [comment]
install worked, notebook opened, but I don't have a muse 2016 to test with battery today, muse 2 didn't connect in the viewer

File: notebooks/cueing.ipynb [hunk 2]
```diff
@@ -140,7 +140,7 @@
     "\n",
     "Run the following to start the viewer and see the raw EEG data stream.\n",
     "\n",
-    "`source activate nbmac`   \n",
+    "`conda activate nbmac`   \n",
```
Comment (kylemath) [inline notebooks/cueing.ipynb:127]: new way of activating conda environment

**Comment by JohnGriffiths** [comment]
@kylemath is this ready to merge now?

**Comment by kylemath** [comment]
yup @JohnGriffiths , just tested, fixed the muse-lsl viewer since default crashes new macOS, and the cueing task works on a mac, n170 didn't see to work, haven't tested others

**Pull request merged**